src/app/frontend/deploy/deployfromsettings.html

```diff
@@ -81,37 +86,38 @@
 
 <kd-help-section>
   <md-checkbox ng-model="ctrl.isExternal" class="md-primary"
-             ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
+               ng-model-options="{ debounce: { 'default': 500, 'blur': 0 } }">
     Expose service externally
   </md-checkbox>
 </kd-help-section>
 
+<!-- advanced options -->
 <div ng-show="ctrl.isMoreOptionsEnabled()">
   <kd-help-section>
-    <md-input-container>
+    <md-input-container class="md-block kd-deploy-block">
```

Comment:
All styles that are defined in this page should be `kd-deploy-form-FOO`. If we decide to reuse them in the future, then we can extract them and rename. Such cases happened in the past. But until this happen, I'd be reluctant to over-generalize for the future. Think abut today and future you've planned. If there's only use case for the class here, make it local here.
Does this make sense?